apps/Arm/vm_qemu_virtio/rpi4/devices-insecure.camkes

```diff
@@ -0,0 +1,151 @@
+            // UART 0 pinctrl
+            /*
+            { "path": "/soc/gpio@7e200000/bt_pins" },
+            { "path": "/soc/gpio@7e200000/uart0_pins" },
+            { "path": "/soc/gpio@7e200000/uart0_gpio14" },
+            */
+            { "path": "/scb/ethernet@7d580000" },
+            // Communication to VC
+            { "path": "/soc/mailbox@7e00b880" },
+            { "path": "/soc/firmware" },
+            // Required by HDMI driver
+            { "path": "/soc/dma-ranges" },
+            { "path": "/soc/dma@7e007000" },
+            { "path": "/soc/interrupt-controller@7ef00100" },
+            // HDMI0
+            { "path": "/soc/hdmi@7ef00700" },
+            { "path": "/soc/i2c@7ef04500" },
+            { "path": "/soc/pixelvalve@7e20a000" },
+            // HDMI1
+            { "path": "/soc/hdmi@7ef05700" },
+            { "path": "/soc/i2c@7ef09500" },
+            { "path": "/soc/pixelvalve@7e216000" },
+            // GPU
+            { "path": "/soc/hvs@7e400000" },
+            { "path": "/soc/txp@7e004000" },
+            { "path": "/gpu" },
+            // Required by 3D acceleration
+            { "path": "/soc/cprman@7e101000" },
+            { "path": "/soc/watchdog@7e100000" },
+            { "path": "/v3dbus/v3d@7ec04000" },
+            // PCIe
+            { "path": "/scb/pcie@7d500000" },
+            { "path": "/scb/pcie@7d500000/pci@0,0" },
+            // USB
+            // TODO: Here need to be following line,
+            // but build system are not able to proceess it
+            // So it will be added in fdt_manipulation.c
+            // Ticket to track: HYPR-434
+            // { "path": "/scb/pcie@7d500000/pci@0,0/usb@0,0" },
+            { "path": "/soc/firmware/reset" },
+            // ramoops memory
+            { "path": "/reserved-memory"},
+        ]);
+
+        vm1.linux_address_config = {
+            "linux_ram_base" : VAR_STRINGIZE(VM1_RAM_BASE),
+            "linux_ram_paddr_base" : VAR_STRINGIZE(VM1_RAM_BASE),
+            "linux_ram_size" : VAR_STRINGIZE(VM1_RAM_SIZE),
+            "linux_ram_offset" : VAR_STRINGIZE(VM_RAM_OFFSET),
+            "dtb_addr" : VAR_STRINGIZE(VM1_DTB_ADDR),
+            "initrd_max_size" : VAR_STRINGIZE(VM_INITRD_MAX_SIZE),
+            "initrd_addr" : VAR_STRINGIZE(VM1_INITRD_ADDR),
+        };
+        vm1.linux_image_config = {
+            "linux_bootcmdline" :
+                "console=hvc0 "
+                "earlycon=pl011,mmio32,0x09000000 "
+                "root=/dev/vda2 "
+                "rw "
+                "debug "
+                "loglevel=8 "
+                "pci=nomsi "
+                "initcall_blacklist=clk_disable_unused ",
+            "linux_stdout" : "hvc0",
+            "dtb_name" : "",
+            "initrd_name" : "",
+        };
+        vm1.mmios = [
+            "0xff846000:0x1000:12"
+        ];
+        /* Note that RAM is not specified here */
+        vm1.irqs =  [];
+        vm1.dtb = dtb([{}]);
+    }
```

Comment:
Initially I did it that way because I thought it is not possible to import CMake or preprocessor variables into CAmkES files, but luckily turns out I was wrong. Now it is done the way you suggest.